pkg/controllers/managementuser/rbac/crtb_handler.go

```diff
@@ -151,10 +237,12 @@ func (c *crtbLifecycle) reconcileCRTBUserClusterLabels(binding *v3.ClusterRoleTe
 			return err
 		})
 		if retryErr != nil {
+			addCondition(binding, condition, FailedToUpdateClusterRoleBindings, binding.Name, retryErr)
 			returnErr = multierror.Append(returnErr, retryErr)
```

Comment:
This could use https://pkg.go.dev/errors#Join too?

Reply:
See above.

Reply:
Done with commit d721dbe69dc5b137ad49d9ff68496e828584e986